### DISCUSSION TITLE
Pesan tidak terkirim Integrasi WA dengan GFORM / Trigger tingkat eror 100%

### DISCUSSION THREAD

**Discussion by galangex** [kirimwa-id/forum#102] [Tanya Jawab]
apakah ada yang salah pada penulisan skrip di google form nya, padahal:
- YOUR_API_TOKEN sudah saya ganti dengan token saya
- YOUR_DEVICE_ID sudah saya ganti sesuai device milik saya,
- YOUR_ADMIN_PHONE_NUMBER sudah saya ganti dengan nomer susai format yang diterima API KirimWA.id
- Trigger juga sudah saya aktifkan,

akan tetapi ketika uji coba dan sudah  submit form, pesan tidak terkirim ke wa penerima dan juga ke wa admin.

ini skrip di google form nya, untuk form ada beberapa kolom, ada nama, nomor whatsapp, ukuran kaos, dll.
![kirim wa](https://user-images.githubusercontent.com/100219787/156088556-e5e74ef8-f6bd-463f-bed0-adeca691946e.jpg)

dan ini triggernya, setelah saya coba isi form, disana muncul tingkat eror 100%
![kirim wa 2](https://user-images.githubusercontent.com/100219787/156088606-e4a009bd-35a5-45a4-9c8d-fbb5f13a1f9c.jpg)

setelah diuj coba, pesan tidak masuk, yang masuk hanya copy respon ke email (saya aktifkan autocopy respon ke email penerima)

kira-kira kesalahan ada di bagian mana ya?
- saya sudah lakukan pairing device ke API KirimWA.id, dan sukses melakukan send message via terminal serta cek status pengiriman message has been sent, mohon bantuannya, terimakasih

**Comment by rioastamal** [comment]
Pesan error dari Google Form seperti apa?

**Reply by adilebang** [reply]
![image](https://user-images.githubusercontent.com/106504602/179928963-265daaa8-f00b-456a-bafe-a97615a39747.png)
Tampilannya begini

**Reply by adilebang** [reply]
Mohon bantuannya bang

**Reply by kafaa-dev** [reply]
@adilebang kodenya mana mas

**Reply by kafaa-dev** [reply]
`device_id` pada payload harus berupa string dan pada kode tersebut menuliskannya tanpa kutip `"` sehingga akan error variable not defined

**Reply by endrianur** [reply]
@syahid246 sdh seperti itu tapi tetap tidak ada pesan wa yanh masuk baik ke admin maupun ke customer.ada solusinya?

**Comment by kafaa-dev** [comment]
`device_id` pada payload harus berupa string dan pada kode tersebut menuliskannya tanpa kutip `"` sehingga akan error variable not defined

**Comment by adelmarz** [comment]
Untuk error code seperti ini, solusinya apa ya? codenya kurang lebih sama seperti diatas
'
![image](https://user-images.githubusercontent.com/121087671/208615082-e0a0462a-02bb-496c-bdae-31e57430759e.png)

**Reply by endrianur** [reply]
saya juga menghadapi masalah yang sama.Kira2 solusinya apa ya?

**Reply by endrianur** [reply]
@rioastamal solusinya seperti apa ya?

**Reply by rioastamal** [reply]
Paste codenya secara lengkap, jangan lupa diformat biar rapi dan gampang dilihat. Sembunyikan informasi yang bersifat sensitif.

**Reply by rioastamal** [reply]
Kalau perlu buat saja thread baru dengan mengikuti kaidah di #2
